src/components/Inputs/WeatherInput.jsx

```diff
@@ -1,53 +1,45 @@
-import './inputs.scss';
-import { useState, useContext } from 'react';
-import { FirstContext } from '../../contexts/FirstContext';
+import '../Tables/tables.scss';
+import { useContext } from 'react';
+import { InputContext } from '../../contexts/InputContext';
+import Input from './Input';
 
 const WeatherInput = () => {
-  const { weatherValues, inputWeatherValue } = useContext(FirstContext);
+	const { date, precip, wind, submitDate, submitPrecip, submitWind } =
+		useContext(InputContext);
 
-  const [weekday, setWeekday] = useState('');
-  const [precip, setPrecip] = useState(null);
-  const [wind, setWind] = useState(null);
+	return (
+		<div className='table-input-row'>
+			<div className='input-cells'>
```

Comment:
For next week can we update the data in our context to be in the form:
```
inputs = {
  precip: { type: 'date', value, isRequired, setValue, key },
  wind, ...
  date, ...
  ...
}
```

So we can turn this entire file to:

```
const { inputs } = useContext(InputContext);
  return (
    <div className='table-input-row'>
      {inputs.map((inputValues) => <div className='input-cells'><Input {...createInputProps(inputValues)} /></div>}
    </div>
  );
```